tools/cdk-build-tools/package.json

```diff
@@ -41,24 +41,24 @@
     "pkglint": "0.27.0"
   },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.15.1",
-    "@typescript-eslint/parser": "^4.15.1",
-    "awslint": "1.91.0",
+    "@typescript-eslint/eslint-plugin": "^4.16.1",
+    "@typescript-eslint/parser": "^4.16.1",
+    "awslint": "1.94.1",
     "colors": "^1.4.0",
-    "eslint": "^7.20.0",
+    "eslint": "^7.21.0",
     "eslint-import-resolver-node": "^0.3.4",
     "eslint-import-resolver-typescript": "^2.4.0",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.3.2",
     "fs-extra": "^9.1.0",
     "jest": "^26.6.3",
-    "jsii": "^1.25.0",
-    "jsii-pacmak": "^1.25.0",
+    "jsii": "^1.24.0",
```

Comment:
Downgrading jsii?

Reply:
Good catch.. I was blindly copying over the [changes in the CDK repo](https://github.com/aws/aws-cdk/compare/v1.91.0...v1.94.1#diff-e3c1b6fc9c18284712d1ca19de6e763a7000d977b15563e8477fdba73b194561R54-R55). Reverted this.